MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerResearchStack.java

```diff
@@ -50,6 +50,8 @@
 
     public MPowerResearchStack(Context context) {
 
+        MPowerPrefs.init(context);
```

Comment:
Can you override the sign out method in MPowerDataProvider to clear these prefs?